lib/assets/javascripts/turbograft/page.coffee

```diff
@@ -19,9 +19,12 @@ Page.refresh = (options = {}, callback) ->
   if options.response
     onlyKeys   = options.onlyKeys   || []
     exceptKeys = options.exceptKeys || []
-    Turbolinks.loadPage null, options.response, true, callback, onlyKeys, exceptKeys
+    Turbolinks.loadPage null, options.response, true, true, callback, onlyKeys, exceptKeys
```

Comment:
Shouldn't this be false so that we _don't_ update the push state when we have the response already?